backstopper-jaxrs/src/main/java/com/nike/backstopper/handler/jaxrs/JaxRsApiExceptionHandler.java

```diff
@@ -0,0 +1,116 @@
+package com.nike.backstopper.handler.jaxrs;
+
+import com.nike.backstopper.apierror.ApiError;
+import com.nike.backstopper.apierror.projectspecificinfo.ProjectApiErrors;
+import com.nike.backstopper.handler.*;
+import com.nike.backstopper.handler.jaxrs.config.JaxRsDefaultExceptionHandlers;
+import com.nike.backstopper.handler.listener.ApiExceptionHandlerListener;
+import com.nike.backstopper.model.DefaultErrorContractDTO;
+import com.nike.backstopper.model.util.JsonUtilWithDefaultErrorContractDTOSupport;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+import javax.ws.rs.ext.Provider;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * An {@link ApiExceptionHandlerServletApiBase} extension that hooks into JAX-RS via its
+ * {@link ExceptionMapper} implementation, specifically {@link ExceptionMapper#toResponse(Throwable)}.
+ *
```

Comment:
Might be good to add to the javadoc comment here that this should be the *only* `ExceptionMapper` registered and used in the application - i.e. it should handle *all* errors.